src/entities/Badges/utils.ts

```diff
@@ -108,7 +104,23 @@ export async function getLandOwnerAddresses(): Promise<string[]> {
     const landOwnersAddresses = new Set(Object.values(landOwnersMap).map((landOwner) => landOwner.owner.toLowerCase()))
     return Array.from(landOwnersAddresses)
   } catch (error) {
-    ErrorService.report("Couldn't fetch land owners", { error, category: ErrorCategory.Badges })
+    ErrorClient.report("Couldn't fetch land owners", { error, category: ErrorCategory.Badges })
     return []
   }
 }
+
+export async function mintBadge(badgeCid: string) {
```

Comment:
this should be `mintBadgeSpec` right?

Reply:
Decided for uploadBadgeSpec and createBadgeSpec